src/App.tsx

```diff
@@ -1,24 +1,148 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import * as todoService from './api/todos';
+import { Header } from './componens/Header/Header';
+import { TodoList } from './componens/Todos/TodoList';
+import { Footer } from './componens/Footer/Footer';
+import { useAppDispatch, useAppSelector } from './app/hook';
+import * as todosAction from './features/todos';
+import * as statusAction from './features/status';
 
-const USER_ID = 0;
+const USER_ID = 11582;
 
 export const App: React.FC = () => {
+  const todos = useAppSelector(state => state.todos.todos);
+
+  const errorMessage = useAppSelector(state => state.todos.errorMessage);
+
+  const dispatch = useAppDispatch();
+
+  const completedTodos = todos.filter(item => item.completed);
+
+  const statusAllCompleted = useAppSelector(
+    state => state.status.statusAllCompleted,
+  );
+
+  useEffect(() => {
+    if (completedTodos.length === todos.length) {
+      dispatch(statusAction.setStatusAllCompleted(true));
+    } else {
+      dispatch(statusAction.setStatusAllCompleted(false));
+    }
+  }, [completedTodos.length]);
```

Comment:
```suggestion
  }, [completedTodos.length, todos.length, dispatch]);
```
fix such linter warnings for all cases
![Screenshot from 2023-10-13 20-34-56](https://github.com/mate-academy/react_todo-app-with-api/assets/33029864/e9d04402-3763-4b06-b90d-568cacaa555e)